flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1173,26 +1275,66 @@ def test_create_sparse(self):
         """
         ``create_volume`` creates sparse files.
         """
-        api = loopbackblockdeviceapi_for_test(test_case=self)
         # 1GB
         apparent_size = REALISTIC_BLOCKDEVICE_SIZE
-        volume = api.create_volume(
+        volume = self.api.create_volume(
             dataset_id=uuid4(),
             size=apparent_size
         )
-        backing_file = api._root_path.descendant(
-            ['unattached', volume.blockdevice_id]
+        size = get_size_info(self.api, volume)
+
+        self.assertEqual(
+            (0, apparent_size),
+            (size.actual, size.reported)
         )
-        # Get actual number of 512 byte blocks used by the file.
-        # See http://stackoverflow.com/a/3212102
-        actual_size = os.stat(backing_file.path).st_blocks * 512
-        reported_size = backing_file.getsize()
 
+    def test_resize_grow_sparse(self):
```

Comment:
Not sure why you care? Presumably if this weren't the case everyone's tests would blow up. But already written so OK.